ShinyModule.R

```diff
@@ -2,24 +2,39 @@ library("shiny")
 library("shinydashboard")
 
 # to display messages to the user in the log file of the App in MoveApps
-# one can use the function from the src/common/logger.R file:
-# logger.fatal() -> logger.trace()
-
-shinyModuleUserInterface <- function(id, label, year) {
-  ns <- NS(id) ## all IDs of UI functions need to be wrapped in ns()
+# one can use the function from the logger.R file:
+# logger.fatal(), logger.error(), logger.warn(), logger.info(), logger.debug(), logger.trace()
 
+shinyModuleUserInterface <- function(id, label, sdk, year) {
+  # all IDs of UI functions need to be wrapped in ns()
+  ns <- NS(id) 
+  # showcase to access a file ('auxiliary files') that is 
+  # a) provided by the app-developer and 
+  # b) can be overridden by the workflow user.
+  fileName <- paste0(getAppFilePath("yourLocalFileSettingId"), "sample.txt")
```

Comment:
I included a showcase how to use the "local file upload" feature.

<img width="915" alt="image" src="https://github.com/movestore/Template_R_Shinydashboard_App/assets/868171/1c2c4458-a0c9-4ee7-a249-d62a0829f591">

Is this a good idea? If yes we should do the same for the _Shiny_ template, too.

Reply:
removed app-settings (was fully replaced by shiny bookmark!)

Reply:
Showcase "local file upload" is now:

![image](https://github.com/movestore/Template_R_Shinydashboard_App/assets/868171/1060db12-e74a-4f0b-bd6c-f0c3694f5295)